cli/cmd/install.go

```diff
@@ -148,7 +152,7 @@ A full list of configurable values can be found at https://artifacthub.io/packag
 			if crds {
 				// The CRD chart is not configurable.
```

Comment:
Not strictly about the changes at hand, but in keeping with the [Boy Scout Rule](https://97-things-every-x-should-know.gitbooks.io/97-things-every-programmer-should-know/content/en/thing_08/), this isn't (strictly speaking) accurate any more [ref](https://github.com/linkerd/linkerd2/blob/main/charts/linkerd-crds/values.yaml#L1).

Should we remove the comment and/or todo?

Reply:
Good point, but let's keep unrelated improvements in their own PRs.